packages/components/combobox/src/tests/multipleSelection.test.tsx

```diff
@@ -373,7 +373,7 @@ describe('Combobox', () => {
       expect(getItem('Pride and Prejudice')).toHaveAttribute('aria-selected', 'true')
     })
 
-    describe('blur behaviour', () => {
+    describe('blur-sm behaviour', () => {
```

Comment:
Probably a search and replace, it is not related to a tailwind className, should remain "blur"